MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -1,72 +1,122 @@
 import UIKit
 
-final class MovieQuizViewController: UIViewController {
+final class MovieQuizViewController: UIViewController, QuestionFactoryDelegate, AlertPresenterDelegate {
+    
+    @IBOutlet private weak var imageView: UIImageView!
+    @IBOutlet private weak var textLabel: UILabel!
+    @IBOutlet private weak var counterLabel: UILabel!
+    @IBOutlet weak var noButton: UIButton!
+    @IBOutlet weak var yesButton: UIButton!
+    
+    private var currentQuestionIndex: Int = 0
+    private var correctAnswers: Int = 0
+    
+    private let questionsAmount: Int = 10
+    private var questionFactory: QuestionFactoryProtocol?
+    private var currentQuestion: QuizQuestion?
+    private var alertPresenter: AlertPresenterProtocol? = nil
```

Comment:
Не обязательно инициализировать здесь
```suggestion
    private var alertPresenter: AlertPresenterProtocol? 
```

Reply:
Исправил